src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -131,6 +132,11 @@ public boolean hasPerson(Person person) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public boolean hasIngredient(Ingredient ingredient) {
+            return false;
```

Comment:
Incorrect test case? Should these be all throwing assertion errors instead? They should not be called as it is a default model stub that have all of the methods failing.